### ISSUE TITLE
rand_indx = torch.tensor(np.random.choice(np.arange(adj_label.shape[0]), batch_size)).type(torch.long) ？ what is adj_label？

### ISSUE THREAD

**Issue by ghost** [kickingass/Graph-MLP#3]

**Comment by ghost** [comment]
`rand_indx = torch.tensor(np.random.choice(np.arange(adj_label.shape[0]), batch_size)).type(torch.long)
`
训练数据包括验集数据和测试集数据？

**Comment by yanghu819** [comment]
adj_label是adj的邻接信息不是ground truth，没有用到val or test的node class label。

**Comment by yanghu819** [comment]
麻烦你看明白transductive node classification的设定， 用全部的Adj和train 的node feature作为train，建议参考tkipf的代码搞明白基础
